packages/gestalt-design-tokens/tokens/classic/comp-web-color-dark.json

```diff
@@ -428,6 +428,22 @@
           "selected": {
             "value": "#ababab"
           }
+        },
+        "hover": {
+          "selected": {
+            "value": "#E7E6DF"
```

Comment:
where are all this color coming from?

Reply:
Are all this these tokens?

Reply:
![Screenshot by Dropbox Capture](https://github.com/user-attachments/assets/66c0b61d-4faf-4379-97f5-60aeab3f0f04)

Reply:
From the token translation guide, I couldn't get the dark mode tokens from Figma, but I confirmed they were correct by looking at the component in the playground mode